### PR TITLE
Updated translations 

### PR DESCRIPTION
**Pull request by MartinJakomin**
Added translations for forms, profile and error messages.

### CODE REVIEW

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -5,125 +5,234 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: piplmesh\n"
-"Report-Msgid-Bugs-To: https://github.com/wlanslovenija/PiplMesh/\n"
-"POT-Creation-Date: 2012-04-19 09:54+0200\n"
-"PO-Revision-Date: 2012-03-21 12:18+0100\n"
-"Last-Translator: Aljaz Prijatelj <aljaz.prijatelj@gmail.com>\n"
-"Language: sl\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Please keep URL to `https://github.com/wlanslovenija/PiplMesh` here.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -5,125 +5,234 @@
...
-#: account/form_fields.py:11
+#: .\account\fields.py:14
 msgid "Value is out of bounds."
 msgstr "Vrednost je izven meja."
 
-#: account/forms.py:42
-msgid "E-mail"
-msgstr "E-mail"
+#: .\account\fields.py:15
+msgid "Callable did not return datetime.date or datetime.datetime object."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Yes, good. Please do not translate internal errors. It is much easier to debug like that. :-)

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -5,125 +5,234 @@
...
+msgstr ""
+
+#: .\account\forms.py:40
+msgid "Username"
+msgstr "Uporabniško ime"
+
+#: .\account\forms.py:44
+msgid "Required. Minimal of 4 characters and maximum of 30. Letters, digits and @/./+/-/_ only."
+msgstr "Potrebno. Minimalno 4 znake in maksimalno 30 znakov. Samo črke, številke in @/./+/-/_ ."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Potrebno_ -> _Obvezno_. _Potrebno_ has a bit strange connotation. :-)

Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Minimalno_? _Najmanj_ is better Slovenian word. :-)

_Najmanj 4 in največ 30 znakov._

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -5,125 +5,234 @@
...
+
+#: .\account\forms.py:46
+msgid "This value may contain only letters, numbers and @/./+/-/_ characters."
+msgstr "To polje lahko vsebuje le črke, številke in znake: @/./+/-/_ ."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Without `:`.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -5,125 +5,234 @@
...
+msgstr "Geslo"
+
+#: .\account\forms.py:71
+msgid "Password (repeat)"
+msgstr "Geslo (ponovi)"
 
-#: account/forms.py:43
+#: .\account\forms.py:73
+msgid "Enter the same password as above, for verification."
+msgstr "Za potrditev vnesite enako geslo kot prej."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _verification_ != _potrditev_

I would use _vnesi_ (we use _ponovi_ above). We want to be more personal.

_V izogib napakam ponovno vnesi isto geslo kot zgoraj._

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -5,125 +5,234 @@
...
+msgid "The two password fields did not match."
+msgstr "Gesli se ne ujemata."
+
+#: .\account\forms.py:93
+msgid "Current password"
+msgstr "Trenutno geslo"
+
+#: .\account\forms.py:95
+msgid "Enter your current password, for verification."
+msgstr "Za potrditev vnesite trenutno geslo."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _vnesi_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -5,125 +5,234 @@
...
+
+#: .\account\forms.py:109
+msgid "Your current password was incorrect."
+msgstr "Vaše trenutno geslo je nepravilno."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Tvoje_

_je bilo nepravilno_

Hm, _Vnešeno geslo je bilo napačno._ ? _nepravilno_?

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -5,125 +5,234 @@
...
+#: .\account\signals.py:20
 msgid "You have been successfully logged out."
 msgstr "Odjava uspešna."
 
-#: account/templates/registration/login.html:5
-#: account/templates/registration/login.html:17
-#: frontend/templates/header.html:43
+#: .\account\views.py:90
+msgid "Registration has been successful."
+msgstr "Registracija uspela."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Registracija je bila uspešna._

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -5,125 +5,234 @@
...
+
+#: .\account\views.py:115
+msgid "Your account has been successfully updated."
+msgstr "Vaš račun je bil uspešno posodobljen."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Tvoj_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -5,125 +5,234 @@
...
+
+#: .\account\views.py:146
+msgid "Your password has been successfully changed."
+msgstr "Vaše geslo je bilo uspešno posodobljeno."
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: _Tvoje_

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -5,125 +5,234 @@
...
+
+#: .\frontend\templates\user\user.html.py:27
+msgid "Last login:"
+msgstr "Zadnja prijava:"
+
+#: .\frontend\templates\user\user.html.py:29
+msgid "Facebook link"
+msgstr "Facebook povezava"
+
+#~ msgid "Reset"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Delete this one. It is redundant now.

File: piplmesh/locale/sl/LC_MESSAGES/django.po (hunk 1 above)
Comment (MartinJakomin) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: Didn't even noticed that poedit has changed the header.

**Comment by MartinJakomin** [comment]
Updated.

**Pull request merged**